<launch>
   <node name="telosb_node" pkg="telosb" type="telosb_node" output="screen"/>    
</launch>
